### PR TITLE
Update p_a_c_k_e_r_unpacker.js

### PR DESCRIPTION
**Pull request by WonderRat**
Fix https://github.com/beautify-web/js-beautify/issues/1111

### CODE REVIEW

**Comment by WonderRat** [comment]
Nah, i'm dummy and don't use Python.

File: js/lib/unpackers/p_a_c_k_e_r_unpacker.js [hunk 1]
```diff
@@ -80,4 +80,4 @@ var P_A_C_K_E_R = {
     }
 
 
-};
\ No newline at end of file
+};
```
Comment (bitwiseman) [inline js/lib/unpackers/p_a_c_k_e_r_unpacker.js]: Remove the newline at the end of the file.

**Comment by bitwiseman** [comment]
@WonderRat - I don't use Python either, but muddle through it for this project. 😄 

Okay, I've looked at https://github.com/beautify-web/js-beautify/blob/master/python/jsbeautifier/unpackers/packer.py

It is already out of sync with the JavaScript version.  Just remove the newline from the end of the file.

File: js/lib/unpackers/p_a_c_k_e_r_unpacker.js (hunk 1 above)
Comment (WonderRat) [inline js/lib/unpackers/p_a_c_k_e_r_unpacker.js]: I think online editor inserts it: http://i.imgur.com/7bJeVNf.png
Its not binary or text file with very strict structure, so i think its not a big deal.

**Pull request closed without merging**